test_spec_NONHEAD_REGRESS

```diff
@@ -0,0 +1,8 @@
+# Includes
+acisfp_check
+dpa_check
+dea_check
+psmc_check
+starcheck
+# Excludes
+-*head*
```

Comment:
This glob should be `*_head*` or maybe even `*_head.py`?

Reply:
And rename to `test_spec_SKA3_REGRESS`.  (Making `HEAD` be an opt-in instead of the default).

Reply:
What do you mean by making 'HEAD' an opt-in and how did you want to effect that change?

Reply:
Opt-in meaning that the default tests like `test_spec_SKA3` or `test_spec_SKA3_REGRESS` should pass on any supported platform , while `test_spec_SKA3_HEAD` would require being on HEAD to pass.  The effect-ation is via adding HEAD to the spec file name, though one could certainly imagine some way to do this within the scripts themselves.  But that's a different story.